src/index.ts

```diff
@@ -326,17 +340,16 @@ export class Intel implements IIntel {
    * @returns A reduced object.
    * @description Get a reduced object which contains profile and statistic data.
    */
-  get() {
+  async get() {
```

Comment:
Is this needed.

Reply:
Yes it is. Without it get won't return a promise.